database-mongo/index.js

```diff
@@ -13,22 +13,83 @@ db.once('open', function() {
 });
 
 var games = mongoose.Schema({
-  token: String,
-  players: String,
-  results: String  
+  token: {type: String, required: true},	
+  playerRoles: String,
+  results: String, 
+  playerIds: String,
 });
 
-var Game = mongoose.model('Game', games);
+module.exports.Game = mongoose.model('Game', games);
 
-var selectGame = function(token, callback) {
-  Game.find({token});
+module.exports.selectGame = function(token, callback) {
+  Game.find({token}, (err, game) => {
+  	console.log(game);
+  	callback(game);
+  });
 };
 
-var createGame = function(token, players, results, callback) {
+module.exports.createGame = function(token, hostName, clientId, callback) {
+	Game.create({token}, (err) => {
+		if (err) {
+			callback(err);
+		} else {
+			var playerRoles = {};
+			playerIds[hostName] = clientId;
+			var playerIds = JSON.stringify(playerIds);
```

Comment:
I'm puzzled. Where are playerIds coming from at line 37 and why does line 38 stomp that?